CPAC/resources/configs/pipeline_config_default.yml

```diff
@@ -1491,6 +1493,8 @@ post_processing:
 
   z-scoring:
 
+    run: On
```

Comment:
Did you mean to turn `spatial_smoothing` and `z-scoring` on in the default pipeline?

Reply:
Yes because I believe they are on in the default pipeline. We just added the `run: ` argument so that it can be turned off if needed. The `run: ` argument wasn't there before.